src/components/admin/RequestContainerBox.jsx

```diff
@@ -28,9 +27,9 @@ const RequestContainerBox = React.forwardRef(
           isLoading={isLoading}
           error={error}
         >
-          {isLoading && <Loader />}
+          {(isLoading || isFetching) && <Loader></Loader>}
```

Comment:
로더 안에 별다른 내용이 없다면 이렇게 작성할 수 있을 것 같아요!
```suggestion
          {(isLoading || isFetching) && <Loader/>}
```